internal/cmd/volume/backup/create/create_test.go

```diff
@@ -0,0 +1,305 @@
+package create
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackitcloud/stackit-cli/internal/cmd/params"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/google/uuid"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+type testCtxKey struct{}
+
+var (
+	testCtx        = context.WithValue(context.Background(), testCtxKey{}, "foo")
+	testClient     = &iaas.APIClient{}
+	testProjectId  = uuid.NewString()
+	testSourceId   = uuid.NewString()
+	testName       = "my-backup"
+	testLabels     = map[string]string{"key1": "value1"}
+	testSourceType = "volume"
+)
+
+func fixtureFlagValues(mods ...func(flagValues map[string]string)) map[string]string {
+	flagValues := map[string]string{
+		globalflags.ProjectIdFlag: testProjectId,
+		sourceIdFlag:              testSourceId,
+		sourceTypeFlag:            "volume",
```

Comment:
```suggestion
		sourceTypeFlag:            testSourceType,
```
or did I get this wrong? :smile: